.travis.xacro_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
As an alternative you could run xacro directly: `/opt/ros/$ROS_DISTRO/lib/xacro/xacro`